pkgs/development/python-modules/triton/default.nix

```diff
@@ -69,20 +69,32 @@ buildPythonPackage {
       })
     ];
 
-  postPatch = ''
+  postPatch =
     # Use our `cmakeFlags` instead and avoid downloading dependencies
     # remove any downloads
-    substituteInPlace python/setup.py \
-      --replace-fail "get_json_package_info(), get_pybind11_package_info()" ""\
-      --replace-fail "get_pybind11_package_info(), get_llvm_package_info()" ""\
-      --replace-fail 'packages += ["triton/profiler"]' ""\
-      --replace-fail "curr_version != version" "False"
-
+    ''
+      substituteInPlace python/setup.py \
+        --replace-fail "get_json_package_info(), get_pybind11_package_info()" ""\
+        --replace-fail "get_pybind11_package_info(), get_llvm_package_info()" ""\
+        --replace-fail 'packages += ["triton/profiler"]' ""\
+        --replace-fail "curr_version != version" "False"
+    ''
     # Don't fetch googletest
-    substituteInPlace unittest/CMakeLists.txt \
-      --replace-fail "include (\''${CMAKE_CURRENT_SOURCE_DIR}/googletest.cmake)" ""\
-      --replace-fail "include(GoogleTest)" "find_package(GTest REQUIRED)"
-  '';
+    + ''
+      substituteInPlace unittest/CMakeLists.txt \
+        --replace-fail "include (\''${CMAKE_CURRENT_SOURCE_DIR}/googletest.cmake)" ""\
+        --replace-fail "include(GoogleTest)" "find_package(GTest REQUIRED)"
+    ''
```

Comment:
Side note: I don't think this is worth inflating the diff, because the only likely reason we might touch this code in future is by removing the whole block when it's no longer necessary. I hope...

Reply:
Good point, resolved in latest